vcx/libvcx/src/v3/handlers/issuance/states.rs

```diff
@@ -151,6 +161,9 @@ impl From<OfferSentState> for FinishedState {
         FinishedState {
             cred_id: None,
             thread_id: state.thread_id,
+            cred_rev_id: None,
+            rev_reg_id: state.rev_reg_id,
+            tails_file: state.tails_file,
```

Comment:
Maybe it could be more future-messy-proof if we extract all revocations related stuff into separate structure?

Something like
```
pub struct TailsRevocation {
            pub cred_rev_id: Option<String>,
            pub rev_reg_id: Option<String>,
            pub tails_file: Option<String>
}
```
And then use it state machines
```
pub struct FinishedState {
    pub cred_id: Option<String>,
    pub thread_id: String,
    pub revocations_v1: Option<TailsRevocation>
    pub status: Status
}
```
So when in future Anoncreds 2.0 come out, we would have just 1 unused field `revocations_v1` instead of having unused `cred_rev_id `, `rev_reg_id `, `tails_file `

Reply:
@Artemkaaas @mirgee what do you guys think?